.github/workflows/timvx_backend_tests.yml

```diff
@@ -37,7 +38,7 @@ jobs:
           cd opencv
           git config user.email "opencv.ci"
           git config user.name "opencv.ci"
-          git pull -v "https://github.com/${{ env.PR_AUTHOR }}/opencv" "${{ env.SOURCE_BRANCH_NAME }}" --allow-unrelated-histories
+          git pull -v "https://github.com/${{ env.PR_AUTHOR_FORK }}" "${{ env.SOURCE_BRANCH_NAME }}"
```

Comment:
Is this change made for those who have opencv fork but with repo name changed?

Reply:
Yes, we've already met this issue recently.

Reply:
Okay, I will forward this change to my PR of workflow for ARM64 as well.